app/packages/core/src/components/ColorModal/controls/ColorAttribute.tsx

```diff
@@ -78,6 +78,7 @@ const ColorAttribute: React.FC<Prop> = ({ eligibleFields, style }) => {
             onClick={() => setOpen((o) => !o)}
             ref={mRef}
             theme={theme}
+            data-cy="select-attribute"
```

Comment:
same here - can we prefix all `data-cy`s with what parent component they are under?
for example here the parent component could be named custom-colors